tools/protein_properties/test-data/protein_properties_result1.bed

```diff
@@ -0,0 +1,3 @@
+ID	Sequence	MW	IP	gravy	length	instability	monoisotpoic
+sp|Q9LHB9|PER32_ARATH	MNFSYSSLSTWTTLMTLGCLLLHSSISSAQLTPTFYDNTCPSVFTIVRDTIVNELRSDPRIAASILRLHFHDCFVNGCDASILLDNTTSFRTEKDAAPNANSARGFPVIDRMKAAVETACPRTVSCADILTIAAQQAVNLAGGPSWRVPLGRRDSLQAFFALANTNLPAPFFTLPQLKASFQNVGLDRPSDLVALSGGHTFGKNQCQFIMDRLYNFSNTGLPDPTLNTTYLQTLRGQCPRNGNQTVLVDFDLRTPTVFDNKYYVNLKELKGLIQTDQELFSSPNATDTIPLVREYADGTQKFFNAFVEAMNRMGNITPLTGTQGQIRQNCRVVNSNSLLHDVVEIVDFVSSM	38846.6361	6.20635986328	-0.0943181818182	352	31.0301420455	0.0880681818182
```

Comment:
Last column is an excellent compromise :)

søn. 7. jun. 2015 kl. 16.17 skrev Björn Grüning <notifications@github.com>:

> In tools/protein_properties/test-data/protein_properties_result1.bed
> <https://github.com/bgruening/galaxytools/pull/131#discussion_r31880709>:
>
> > @@ -0,0 +1,3 @@
> > +ID	Sequence	MW	IP	gravy	length	instability	monoisotpoic
> > +sp|Q9LHB9|PER32_ARATH	MNFSYSSLSTWTTLMTLGCLLLHSSISSAQLTPTFYDNTCPSVFTIVRDTIVNELRSDPRIAASILRLHFHDCFVNGCDASILLDNTTSFRTEKDAAPNANSARGFPVIDRMKAAVETACPRTVSCADILTIAAQQAVNLAGGPSWRVPLGRRDSLQAFFALANTNLPAPFFTLPQLKASFQNVGLDRPSDLVALSGGHTFGKNQCQFIMDRLYNFSNTGLPDPTLNTTYLQTLRGQCPRNGNQTVLVDFDLRTPTVFDNKYYVNLKELKGLIQTDQELFSSPNATDTIPLVREYADGTQKFFNAFVEAMNRMGNITPLTGTQGQIRQNCRVVNSNSLLHDVVEIVDFVSSM	38846.6361	6.20635986328	-0.0943181818182	352	31.0301420455	0.0880681818182
>
> Yeah thought about this too, but then it's very easy to get the FASTA file
> back (for example after filtering) with tabular-to-faster.
> On the other hand, the user could extract al sequences from the original
> FASTA with the filtered ID's.
> To make it short I'm not sure :) Moving it as last column? Or removing it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bgruening/galaxytools/pull/131/files#r31880709>.
>